lib/mutant/color.rb

```diff
@@ -24,7 +24,7 @@ def format(text)
         text
       end
 
-    private
+    private # rubocop:disable AccessModifierIndentation
```

Comment:
Do you prefer it over the way it is currently written (meaning you therefore prefer just leaving in the `rubocop:disable`)?